src/graphql.tsx

```diff
@@ -191,14 +191,18 @@ export default function graphql(
       return opts;
     }
 
+    function shouldSkip(props) {
+      return mapPropsToSkip(props) || (mapPropsToOptions(props) as QueryOptions).skip;
```

Comment:
Yeah, we should probably just do that once in `willRecieveProps` (and initially).

I wouldn't expect people would have assumptions about how many times it's called though.